packages/manager/cypress/e2e/core/volumes/update-volume.spec.ts

```diff
@@ -64,13 +61,58 @@ describe('volume update flow', () => {
           .should('be.visible')
           .closest('tr')
           .within(() => {
-            cy.findByText('Edit').click();
+            cy.get('button').click();
           });
+        cy.get('[data-testid="Edit"]').click();
 
-        // Confirm new label and tags are shown.
+        // Confirm new label is shown.
         cy.get('[data-qa-drawer="true"]').within(() => {
           cy.findByText('Edit Volume').should('be.visible');
           cy.findByDisplayValue(newLabel).should('be.visible');
+        });
+      }
+    );
+  });
+
+  /*
+   * - Confirms that volume tags can be changed from the Volumes landing page.
+   */
+  it("updates volume's tags", () => {
+    cy.defer(() => createVolume(volumeRequest), 'creating volume').then(
+      (volume: Volume) => {
+        cy.visitWithLogin('/volumes', {
+          // Temporarily force volume table to show up to 100 results per page.
+          // This is a workaround while we wait to get stuck volumes removed.
+          // @TODO Remove local storage override when stuck volumes are removed from test accounts.
+          localStorageOverrides: {
+            PAGE_SIZE: 100,
+          },
+        });
+
+        // Confirm that volume is listed on landing page, click "Edit" to open drawer.
+        cy.findByText(volume.label)
+          .should('be.visible')
+          .closest('tr')
+          .within(() => {
+            cy.get('button').click();
+          });
+        cy.get('[data-testid="Manage Tags"]').click();
```

Comment:
```suggestion
        // Confirm that volume is listed on landing page, click "Edit" to open drawer.
        cy.findByText(volume.label)
          .should('be.visible')
          .closest('tr')
          .within(() => {
            ui.actionMenu
              .findByTitle(`Action menu for Volume ${volume.label}`)
              .should('be.visible')
              .click();
          });

        ui.actionMenuItem
          .findByTitle('Manage Tags')
          .click();
```

We've got a bunch of helpers that make selecting common UI elements like drawers, modals, buttons, action menus, etc., a lot easier. You'll have to import the `ui` object via `import { ui } from 'support/ui'`, but using those should make it a lot easier to write tests without having to go searching through the codebase for test tags (and helps keep hardcoded IDs out of the tests)

Reply:
Hello @jdamore-linode,

Thank you for your comments, everything should be resolved now.